setup.py

```diff
@@ -1,28 +1,72 @@
 #!/usr/bin/env python3
 
+import os
 import sys
 import numpy as np
 
-from setuptools import setup
-from distutils.extension import Extension
-from Cython.Build import cythonize
+from setuptools.extension import Extension
+from Cython.Build import build_ext
+
+dirname = os.path.dirname(os.path.abspath(__file__))
+
+if "--no-build-core" in sys.argv:
```

Comment:
`--no-build-core` -> `--build-core-library` to adjust argument naming with `--bundle-core-library`.

Reply:
I changed it to `--no-build-core-library` because the core library will be built in the default settings for PyPI. If the core library does not exist in the source tree, the module will be built without the core library.